server/src/main/resources/application-e2e.properties

```
@@ -13,3 +13,5 @@ kitu.oppijanumero.password=
 kitu.oppijanumero.callerid=1.2.246.562.24.85478397072
 kitu.oppijanumero.casUrl=https://virkailija.untuvaopintopolku.fi/cas
 kitu.oppijanumero.serviceUrl=https://virkailija.untuvaopintopolku.fi/oppijanumerorekisteri-service
+
+kitu.scheduling.yki.enabled=false
```

voitaisko tehdä näitä Domain Driven development - tyyliin? Eli:
```suggestion
kitu.yki.scheduling.enabled=false
```